dsa/validate.go

```diff
@@ -1,51 +1,104 @@
 package dsa
 
 import (
+	"errors"
+
 	"github.com/prebid/prebid-server/v2/exchange/entities"
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/util/jsonutil"
+)
+
+// ObjectSignal represents publisher DSA object required statuses
+type ObjectSignal int8
```

Comment:
I'm ok with the approach used in the OpenRTB library in general, though I would like to avoid `Required` as a type name and `Required` as a value name resulting in `RequiredRequired Required = 2`.
In that case, I would shy away from using the exact field name as the type as I don't think there is a more fitting value name than `Required` for the value 2. So then I'd say let's make these types symmetric where they all have some suffix like `Signal`:
```
RequiredSignal, PubSignal, BuyerSignal
```
or
```
RequiredSignal, PubRenderSignal, AdRenderSignal
```
or
```
RequiredSignal, PubSignal, AdSignal
````

Thoughts?